blog/2024-01-23/index.md

```diff
@@ -0,0 +1,58 @@
+---
+slug: over-one-million-active-users-and-growing
+title: Over One Million Active Users—and Growing
```

Comment:
Did you mean to use a dash here?

Reply:
Yep, an em-dash to emphasize the aside. It's read similarly to "Over One Million Active Users… and Growing" or "Over One Million Active Users. And Growing".

Reply:
Alternative suggestion:

```suggestion
title: Over One Million Active Users, and Growing
```